public/app/features/explore/Wrapper.tsx

```diff
@@ -1,18 +1,43 @@
-import React, { Component } from 'react';
-import { connect } from 'react-redux';
+import React, { PureComponent } from 'react';
```

Comment:
The change from Component to PureComponent should be fine AFAIK but maybe @ivanahuckova or @aocenas can confirm this too?

Reply:
As far as I know, the change should be okay. 🙂

Reply:
I don't think it should break anything but I have a feeling:
```
navModel: getNavModel(state.navIndex, 'explore'),
```
will break any benefit as it seems to create a new object every time.

Reply:
great 🙌🏾, thank you @ivanahuckova for taking a look, I will proceed to merge :)

Reply:
> I don't think it should break anything but I have a feeling:
> 
> ```
> navModel: getNavModel(state.navIndex, 'explore'),
> ```
> 
> will break any benefit as it seems to create a new object every time.

Hey, @aocenas thanks for checking out this PR 😄 , I was looking into `getNavModel` function, even though it generates a new object every time, the values are always the same if the parameters don't change, and because PureComponent only does a shallow comparison on the `ComponentShouldUpdate` I think will keep the benefits. I also verified the children's `Wrapper` component to confirm they are also PureComponents only to make sure are not breaking anything.

let me know if I am mistaken(I still lacking context in the project 😅 ) , I will merge the PR but ofc, if we find later on this, don't add any benefits we can put it back as before :)